server/routes.js

```diff
@@ -16,7 +16,7 @@ export default function(server) {
   });
 
   server.get(/^\/(images|fonts)\/.*/, function(req, res) {
```

Comment:
That'd be an overkill. This is a backup incase a request for an image is sent directly to our servers. We should never have that.